src/main/java/org/deepsymmetry/beatlink/dbserver/ConnectionManager.java

```diff
@@ -283,6 +302,7 @@ private void requestPlayerDBServerPort(DeviceAnnouncement announcement) {
             os.write(DB_SERVER_QUERY_PACKET);
             byte[] response = readResponseWithExpectedSize(is, 2, "database server port query packet");
             if (response.length == 2) {
+                portFound = true;
```

Comment:
Based on what I found in my own investigation tonight, this would fail with my players, because we do get back a 2 byte response (which would cause `portFound` to be set to `true`) but the value we get back is actually `-1`. So if taking this approach, we would want to test the actual port number in the response. And probably add some sleep time if we get a `-1` back.